python/tvm/relay/testing/init.py

```diff
@@ -160,6 +160,13 @@ def create_workload(net, initializer=None, seed=0):
     for k, v in shape_dict.items():
         if k == "data":
```

Comment:
can we do something like the following and document it?
```python
if k == "data" or any(isinstance(d, tvm.expr.Var) for d in v.shape):
```
so that line 163~169 can be removed?